pkg/virt-handler/device-manager/generic_device.go

```diff
@@ -260,7 +260,7 @@ func (dpi *GenericDevicePlugin) Allocate(ctx context.Context, r *pluginapi.Alloc
 	dev := new(pluginapi.DeviceSpec)
 	dev.HostPath = dpi.devicePath
 	dev.ContainerPath = dpi.devicePath
-	dev.Permissions = "rw"
+	dev.Permissions = "rwm"
```

Comment:
I see that we add `mknod` ability now. Do we need it now for some reason?

Reply:
Ah found the explanation. Should we distinguish between `v1` and `v2` here and in one case add the `m` and in the other case not?

Reply:
I was thinking about it as well. I wanted to avoid having smth like `if v1 {...} else {...}` accross the code. Maybe I could introduce one more function in `cgroup` package which would return the needed permissions for the device... but imho that would look a bit weird. If distinguishing here between `v1` and `v2` I would probably go with the `if` solution as the most straightforward one. Plus some comment in the code why it is needed. At least it will be clear that its a workaround.

And In fact I've created a PR in upstream `runc` to fix the ebpf program. The PR is already merged. With the fix applied the `mknod` permission is not required and the behaviour is the same as with `v1` (so this change becomes redundant). My assumption so far has been that compatibility with older `runc` versions should be preserved. Is it the case here?